data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1667 @@
+			label accept
+			`	Alondo makes a few calls, arranging some meetings to inform Free Worlds officials of what you've told him, then heads out with you and the others. You go to a cozy restaurant near the center of town, where Ijs and Katya tell you a bit of their recent work in the Conservatory, and JJ laments some trouble with handling patrol fleets. Alondo keeps asking you to tell more of your travels with the jump drive, and what you've seen of the galaxy. You leave at dusk, saying your goodbyes to everyone, and heading back to your ship. You should head back to <planet> now and tell House Bebliss that Freya's agreed to their proposal.`
+				accept
+
+mission "House Bebliss 5-FW"
+	name "Jumping Spiders"
+	description "Acquire a spare Jump Drive and bring it to <destination>, where the House Bebliss representatives will have it installed in one of their ships."
+	landing
+	to offer
+		has "House Bebliss 4-FW: done"
+	source "Factory of Eblumab"
+	destination "Ablub's Invention"
+	on offer
+		conversation
+			`You land on the hangars beside House Bebliss' estate, beside an Arach Spindle, and are once again welcomed by maids and butlers, who show you to a room where the same three Arachi from last time are.`
+			`	"Agreed to work with us has she, Captain?" They ask.`
+			`	You tell them that Freya has agreed to work with them, letting them study the Pug artifacts she has collected on Deneb, and explain she first asked you to inform the Free Worlds about the Coalition first.`
+			`	"Understandable, that is. An unusual request, from an unknown group, it was." They have some of the servants start carrying dozens of luggage bags outside. "If in agreement, miss Freya is, head to that planet, 'Deneb,' to meet her, we shall."`
+			choice
+				`	"How do you plan to do that?"`
+				`	"Do you have a Jump Drive?"`
+			`	"From the Heliarchs, much information our House can gather. Much more difficult, however, acquiring a jump drive is. Risky to possess one, even," they explain. "Assist us on that front, you must, Captain. Bring a jump drive, to <destination>, you must. Equip one of our ships with it, we will. <payment>, you will receive."`
+			choice
+				`	"Why can't I just bring you to Deneb myself?"`
+					goto myself
+				`	"You just said it's risky to possess one though."`
+			`	"Indeed. Keep it, we would not," another one responds. "Once concluded, our business with your friend Freya is, and returned, we have, give you back the Jump Drive, we would."`
+				goto accept
+			label myself
+			`	"Stay there for some months, we will, and though agreed to help us, your friend Freya has, troubled by our presence, other humans may be," they respond. "So that issues with accomodating us, she does not have, use our ship as a base, we will."`
```

Comment:
```suggestion
			`	"Stay there for some months, we will, and though agreed to help us, your friend Freya has, troubled by our presence, other humans may be," they respond. "So that issues with accommodating us, she does not have, use our ship as a base, we will."`
```